client/components/basic/PlanTag.js

```diff
@@ -0,0 +1,45 @@
+import React, { useMemo, useState } from 'react';
+import { Tag, Box } from '@rocket.chat/fuselage';
+
+import { useMethod } from '../../contexts/ServerContext';
+
+function PlanTag() {
+	const [plans, setPlans] = useState([]);
+	const [background, setBackground] = useState([]);
+
+	const getTags = useMethod('license:getTags');
+
+	useMemo(() => {
+		const loadTags = async () => {
+			setPlans(await getTags());
```

Comment:
There's a catch here: `setPlans` can be accidentally called *after the component was unmounted*, i.e., `PlanTag` may be already unmounted after the `getTags()` returned promise was resolved. To avoid that we usually do some kind of semaphore to skip the `setPlans` calls after unmount, but I've made a custom hook for that:

```js
import { useSafely } from '@rocket.chat/fuselage-hooks';

const [plans, setPlans] = useSafely(useState([]));
```

So `setPlans` can be called anytime.

N.B.: Pay attention to possible memory leaks, tough. If the promise never resolves, `setPlans` will be referenced forever, and the whole component as well.